rpcs3/Emu/Cell/PPUFunction.cpp

```diff
@@ -347,7 +347,7 @@ extern std::string ppu_get_syscall_name(u64 code)
 	case 573: return "sys_pad_dbg_ldd_set_data_insert_mode";
 	case 574: return "sys_pad_ldd_register_controller";
 	case 575: return "sys_pad_ldd_get_port_no";
-	case 577: return "sys_pad_manager_...";
+	case 579: return "sys_bluetooth_aud_serial_unk1";
```

Comment:
Are you sure the syscall has "unk1" in the name? Seems very unlikely

Reply:
Actually you are right. Fixed.